bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/i18n/androiddebugbridge_it.properties

```
@@ -26,7 +26,7 @@ thing-type.config.androiddebugbridge.android.deviceMaxVolume.description = Presu
 thing-type.config.androiddebugbridge.android.ip.label = Indirizzo IP
 thing-type.config.androiddebugbridge.android.ip.description = Indirizzo IP del dispositivo.
 thing-type.config.androiddebugbridge.android.mediaStateJSONConfig.label = Configurazione Stato Media
-thing-type.config.androiddebugbridge.android.mediaStateJSONConfig.description = Configurazione JSON che permette di modificare la strategia di rilevamento dello stato dei media per ogni app. Fare riferimento alla documentazione del binding.
+thing-type.config.androiddebugbridge.android.mediaStateJSONConfig.description = Configurazione JSON che permette di modificare la strategia di rilevamento dello stato dei media per ogni app. Fare riferimento alla documentazione del addon.
```

This should be have been replaced.
```suggestion
thing-type.config.androiddebugbridge.android.mediaStateJSONConfig.description = Configurazione JSON che permette di modificare la strategia di rilevamento dello stato dei media per ogni app. Fare riferimento alla documentazione del binding.
```